app/RazorPagesContacts.csproj

```diff
@@ -1,17 +1,21 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net5.0</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
-     <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="3.1.0">
+     <!-- To use the 'dotnet-ef' global tool, the project must include the 'Microsoft.EntityFrameworkCore.Design' package.
+          The following lines were added by running: 'dotnet add package Microsoft.EntityFrameworkCore.Design'.
+     -->
+     <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="5.0.0">
        <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
        <PrivateAssets>all</PrivateAssets>
      </PackageReference>
-     <PackageReference Include="Microsoft.EntityFrameworkCore.InMemory" Version="3.1.0" />
-     <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="3.1.0" />
```

Comment:
Is `Microsoft.EntityFrameworkCore.SqlServer` not longer required? No longer supported?

Reply:
It wasn't needed before either.